Source/ASTableNode.mm

```diff
@@ -80,7 +80,7 @@ - (instancetype)initWithStyle:(UITableViewStyle)style
     [self setViewBlock:^{
       // Variable will be unused if event logging is off.
       __unused __typeof__(self) strongSelf = weakSelf;
```

Comment:
Yep but we should strongify once, rather than implicitly strongifying each time we use `weakSelf`.